### ISSUE TITLE
Errors when connecting to Colab KataGo via Lizzie

### ISSUE THREAD

**Issue by foolai** [kinfkong/katago-colab#3]
KataGo and SSH server on Colab server run fine.
However, when connecting Colab KataGo via Lizzie, I get an error.
I searched the internet and couldn't find a solution. Please help me.

(1) Lizzie Engine 0 -> C:/katagocolab/colab-katago.exe testuser 87651234

(2) Two error messages appearing in the gtp console window

case 1

1 name
2 version
3 boardsize 19
4 komi 6.5
2021/01/30 10:11:37 INFO using user name: testuser password: 87651234
2021/01/30 10:11:38 ssh options
{"host": "0.tcp.ngrok.io", "port": 16308, "user": "root"}
2021/01/30 10:11:38 failed to create ssh clientssh: handshake failed: read tcp 122.147.186.214:49253->3.17.7.232:16308: wsarecv: An existing connection was forcibly closed by the remote host.

case 2

1 name
2 version
3 boardsize 19
4 komi 6.5
2021/01/30 10:28:08 INFO using user name: testuser password: 87651234
2021/01/30 10:28:09 ssh options
{"host": "0.tcp.ngrok.io", "port": 16308, "user": "root"}
2021/01/30 10:28:09 failed to create ssh clientssh: handshake failed: EOF

**Comment by foolai** [comment]
I used it for 4 months without any problems. However, an error occurs from yesterday.

**Comment by andyleeyuan** [comment]
I have the same problem

**Comment by wineandgolover** [comment]
I have the same problem with the updated 1.8.1 image. 

I am in the USA, is this code trying to connect to a server in China? (endpoint = 'http://oss-cn-beijing.aliyuncs.com') and could that be a problem? If so, how easy would it be to reproduce the host in the USA?

**Comment by lowrank** [comment]
> I have the same problem with the updated 1.8.1 image.
> 
> I am in the USA, is this code trying to connect to a server in China? (endpoint = 'http://oss-cn-beijing.aliyuncs.com') and could that be a problem? If so, how easy would it be to reproduce the host in the USA?

YES, I have fixed the problem by erasing those codes (https://github.com/lowrank/katago-colab) . But this also changes the argument to the program. 

After compiling, you will get the executable, say name is katago-colab.exe, then  one can use 

`katago-colab.exe hostname password port`

to use. 

For example:

`katago-colab.exe 0.tcp.ngrok.io LONGPASSWORD 12345`

**Comment by Google9876** [comment]
1 name
2 version
3 boardsize 19
4 komi 6.5
2021/01/30 10:28:08 INFO using user name: testuser password: 87651234
2021/01/30 10:28:09 ssh options
{"host": "0.tcp.ngrok.io", "port": 16308, "user": "root"}
2021/01/30 10:28:09 failed to create ssh clientssh: handshake failed: EOF 




You have to run colab again
